### ISSUE TITLE
map transit incidents

### ISSUE THREAD

**Issue by EddyIonescu** [trynmaps/opentransit-map#89]
While many agencies internally record incidents (which can often cause bunching and gaps down the line for hours), the TTC (Toronto Transit Commission) makes then publicly available here: https://www.toronto.ca/city-government/data-research-maps/open-data/open-data-catalogue/#bb967f18-8d90-defc-2946-db3543648bd6

This ticket only tracks putting them on the map (use fake hardcoded data when implementing this)

API endpoint creation tracked here: https://github.com/trynmaps/tryn-api/issues/30

**Comment by lukemccrae** [comment]
I'm looking into this. Should the frontend expect the location data to be geocoded into LatLng points?

**Comment by EddyIonescu** [comment]
@lukemccrae no, you should expect it to be a `{lat: Number, lon: Number}` object, from which you can make the LatLng point yourself.
